pkg/testers/ginkgo/ginkgo.go

```diff
@@ -247,6 +245,7 @@ func NewDefaultTester() *Tester {
 		TestPackageBucket: "kubernetes-release",
 		TestPackageDir:    "release",
 		TestPackageMarker: "latest.txt",
+		Timeout:           86400000000000,
```

Comment:
Any reason not do do:
```suggestion
		Timeout:           24 * time.Hour,
```

Reply:
Didn't think of it. let me do that

Reply:
Fixed